x-pack/plugins/cases/public/containers/mock.ts

```diff
@@ -168,6 +170,25 @@ export const basicResolvedCase: ResolvedCase = {
   aliasTargetId: `${basicCase.id}_2`,
 };
 
+export const basicCaseMetrics: CaseMetricsResponse = {
+  alerts: {
+    count: 12,
+    hosts: {
+      total: 2,
+      values: [
+        { name: 'foo', count: 2 },
+        { name: 'bar', count: 10 },
+      ],
+    },
+    users: {
+      total: 1,
+      values: [{ name: 'Jon', count: 12 }],
+    },
+  },
+  connectors: [{ id: 'none', name: 'My Connector', pushCount: 3 }],
```

Comment:
What this metric is about?

Reply:
No we haven't implemented the connectors metric yet. Our plan is to use this field to determine how many unique connectors have been used within the case. For this metric:

![image](https://user-images.githubusercontent.com/56361221/145874508-a869c3f2-6d25-48ae-90e2-50573ae01caf.png)

aka `connectors.length`

As for the `pushCount` I could have sworn we had a request for number of pushes per connector 🤔  but I can't find it haha. I have it written in the 8.1 planning doc but I don't see it in any of the other issues we have 🤷‍♂️  I'm happy to remove it.

Reply:
if we don't need the `pushCount` maybe we could simplify the `connectors` value ang be more similar to `alerts`:
```
connectors: {
  total: 2
}
```
This way we don't have to return data we don't need, and if one day we need it we can add the `values` (as it is in alerts) without having to change the existing structure.

Reply:
Yeah that's fine with me.

```
connectors: {
  total: number;
}
```

Sounds good 👍

Reply:
Fine by me!